app/config/schema/config.proto

```diff
@@ -0,0 +1,216 @@
+/*
+ * This file defines and documents the format for config.json and other
+ * top-level configuration (base.json, development.json, production.json)
+ *
+ * Proto doc: https://developers.google.com/protocol-buffers/
+ */
+
+syntax = "proto3"; // Must be proto3 to support JSON
+
+message Config {
+    // TODO: What does this flag do?
+    bool debug = 1;
+
+    // The min number of visited rooms before a guest is shown the map
+    // Optional. Defaults to 0.
+    int32 map_unlock_threshold = 2;
+
+    // What base URL to use for iframing Jitsi video calls
+    // Can be overridden by "overrideJitsiServerUrlWithWindowHost" (see below)
+    // Optional. If not set, "overrideJitsiServerUrlWithWindowHost" must be true.
+    string jitsi_server_url = 3;
+
+    // Whether to use the host site for Jitsi call URLs
+    // Overrides "jitsiServerUrl" (see above)
+    // Optional. Defaults to false.
+    bool override_jitsi_server_url_with_window_host = 4;
+
+    // Passed to Jitsi when connecting to determine whether to use SSL.
+    // Optional. Defaults to false.
+    bool no_jitsi_server_SSL = 5;
+
+    // Whether to store state locally in the user's browser.
+    // If false, reloading the page will mean rejoining as a new user.
+    // Optional. Defaults to false.
+    bool use_local_sessions = 6;
+
+    // Options related to space moderators
+    // Optional.
+    Moderation moderation = 7;
+
+    // ID of the room to place users when they first join
+    // Required.
+    string start_room = 8;
+
+    // Default display name in Jitsi video calls
+    // Required.
+    string video_display_name = 15;
+
+    // Options related to username & avatar selection on the welcome page
+    // Required.
+    WelcomePage welcome_page = 9;
+
+    // Sets of avatar choices
+    // Each AvatarSet should have the same number of images
+    // Required if "avatarSelectionEnabled" is true in "welcomePage".
+    repeated AvatarSet avatars = 10;
+
+    // Set of additional avatars for specific situations
+    // Correspond to the user's choice of avatar
+    // Optional
+    SpecialAvatarSets special_avatars = 11;
+
+    // Options related to displaying the map
+    // Required.
+    MapConfig map = 12;
+
+    // Options related to the poking feature
+    // Optional. If not provided, poking is turned off.
+    PokeConfig poke = 13;
+
+    // Number of server-side proxies in use (if unsure, probably 0)
+    // Required.
+    int32 num_proxies = 14;
+}
+
+message Moderation {
+    // Contact emails for moderators to receive moderation form submissions
+    // TODO: Is this currently used?
+    // Required (at least one).
+    repeated string moderator_emails = 1;
```

Comment:
@morganecf @hndrewaall it seems like nothing currently calls the send_email endpoint. Is that accurate? Is that temporary, or has email been deprecated in favor of text? How should I document this?

Reply:
Using text instead of email was a super last minute change (as was doing the whole thing via a subprocess call). Let's keep this documented for now, because we'll want to switch back to email. We also want to keep emails out of the config and set them as env variables or something, since it'd be better not to commit them.